src/amity.py

```diff
@@ -0,0 +1,623 @@
+from random import randint
+
+from db import Db
+from fellow import Fellow
+from fileman import FileMan
+from livingspace import LivingSpace
+from migration import Migration
+from office import Office
+from staff import Staff
+from util import Util
+
+
+class Amity(FileMan):
```

Comment:
**Code Quality:** there is a better way for dependency injection than using  _inheritance_. `Fileman` should be a _util_.